spec/moonshot/dynamic_template_spec.rb

```diff
@@ -4,7 +4,7 @@
   include FakeFS::SpecHelpers
```

Comment:
I think we need a regression test that fails with the old code and passes with the new code.  I'm not sure if this change to the parameters `let` accomplishes that.  Does it?

Reply:
Now it accepts both ways: lambda and string. So both code old and new are accepted.

Reply:
Ok, we need another test then.  We should have a regression test that would have caught the original bug.

Reply:
@pdrakeweb the bug didn't really lie in the plugin itself, but rather in the way we _configured_ it, the incorrect set of values have been passed on. That is a "bug" in the `ControllerConfig` class which we chose not to address (it would have been extremely complicated to tackle the issue), but created a workaround instead. Long story short, I think a regression test is not possible in this case.